packaging/fedora/snapd.spec

```diff
@@ -220,6 +242,7 @@ Provides:      bundled(golang(gopkg.in/yaml.v2))
 %endif
 
 # Generated by gofed
+Provides:      golang(%{import_path}/apparmor) = %{version}-%{release}
```

Comment:
This directory is no longer in the tree.
If this is stale is the rest correct?

Reply:
I can regenerate the Provides for 2.29.2, I just hadn't done it yet. This is from 2.28.5.